polynote-frontend/polynote/main.ts

```diff
@@ -217,4 +217,8 @@ monaco.languages.registerCompletionItemProvider('sql', {
     provideCompletionItems: (doc, pos, context, cancelToken) => {
         return (doc as CodeCellModel).requestCompletion(doc.getOffsetAt(pos));
     }
-});
\ No newline at end of file
+});
+
+document.fonts.ready.then(() => {
```

Comment:
Can you please add a comment linking to https://github.com/microsoft/monaco-editor/issues/648 explaining why this is needed?

Reply:
Ok, done.